### PR TITLE
Add the style cache file as well as associated tests and test files

### PR DESCRIPTION
**Pull request by andrewO423**
This was worked on by @jpvonelli @Meono55 as well as myself.

### CODE REVIEW

File: src/abv/inventory_api/style_cache.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+class StyleCache:
+
+    """Cache constructor."""
+    def __init__(self, filename):
+        self.filename = filename
+        self.cache_dict = {}
+
+        with open(self.filename, 'r') as infile:
+            i = 0
+            for line in infile:
```
Comment (stevemac1900) [inline src/abv/inventory_api/style_cache.py]: There's probably a better way to skip the first line in the for loop, don't know whether it's worth changing at the moment but we should refactor this.

File: src/abv/inventory_api/style_cache.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (stevemac1900) [inline src/abv/inventory_api/style_cache.py]: All of the docstrings in this file are redundant, probably just remove them all.

File: src/abv/inventory_api/style_cache.py (hunk 1 above)
Comment (andrewO423) [reply src/abv/inventory_api/style_cache.py]: Will look into it.

File: src/abv/inventory_api/style_cache.py (hunk 2 above)
Comment (andrewO423) [reply src/abv/inventory_api/style_cache.py]: Agreed.

**Pull request merged**